tests/test_descriptions.py

```diff
@@ -0,0 +1,78 @@
+from ariadne import make_executable_schema
+
+query_definition = '''
+    """
+    A simple GraphQL schema which is well described.
+    """
+    type Query {
+        """
+        Translates a string from a given language into a different language.
+        """
+        translate(
+            "The original language that `text` is provided in."
+            fromLanguage: Language
+
+            "The translated language to be returned."
+            toLanguage: Language
+
+            "The text to be translated."
+            text: String
+        ): String
+    }
+'''
+
+enum_definition = '''
+    """
+    The set of languages supported by `translate`.
+    """
+    enum Language {
+        "English"
+        EN
+
+        "French"
+        FR
+
+        "Chinese"
+        CH
+    }
+'''
+
+
+def test_enums_include_descriptions():
+    schema = make_executable_schema(enum_definition)
+
+    description = schema.get_type("Language").description
+    assert description == "The set of languages supported by `translate`."
+
+
+def test_enum_values_include_descriptions():
+    schema = make_executable_schema(enum_definition)
+
+    for value in schema.get_type("Language").values.values():
+        assert isinstance(value.description, str)
```

Comment:
Could this be replaced with 1 assert explicitly testing one of items for having description?

Loops generate nice coverage, but they provide any feedback if they didn't loop over anything, which is bad. :(